docs/plans/2026-02-16-policy-import-export-plan.md

```
@@ -0,0 +1,879 @@
+    /// Name of this policy preset.
+    pub name: String,
+    /// Author email or identifier.
+    #[serde(default)]
+    pub author: Option<String>,
+    /// Intended use case or scenario.
+    #[serde(default)]
+    pub intent: Option<String>,
+    /// Additional notes or description.
+    #[serde(default)]
+    pub notes: Option<String>,
+    /// When this bundle was created.
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)